sympy/codegen/tests/test_algorithms.py

```diff
@@ -0,0 +1,128 @@
+from __future__ import (absolute_import, division, print_function)
+
+import sys
+import shutil
+
+import sympy as sp
+from sympy.core.compatibility import exec_
+from sympy.codegen.ast import Assignment
+from sympy.codegen.algorithms import newtons_method, newtons_method_function
+from sympy.codegen.fnodes import bind_C
+from sympy.codegen.futils import render_as_module as f_module
+from sympy.codegen.pyutils import render_as_module as py_module
+from sympy.external import import_module
+from sympy.printing.ccode import ccode
+from sympy.utilities._compilation import compile_link_import_strings
+from sympy.utilities.pytest import skip, USE_PYTEST, raises
+
+cython = import_module('cython')
+wurlitzer = import_module('wurlitzer')
```

Comment:
Looks like the test is skipped on travis because this file requires Cython and wurlitzer.

Reply:
I see, the feedstock is empty still: https://github.com/conda-forge/wurlitzer-feedstock

Reply:
Should I pip install wurlitzer in the mean time or should we wait?